src/App.jsx

```diff
@@ -25,21 +26,18 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+
+    <Person
+      person={misha}
+    />
+
+    <Person
+      person={olya}
+    />
+
+    <Person
+      person={alex}
+    />
+
```

Comment:
I think you don't need to separate the Person function component line when you have only a single property. But of course, it is a preference :)

```
<Person person={misha} />
<Person person={olya} />
<Person person={alex} />
```

Reply:
agree with @dolnaro here